src/vs/workbench/contrib/quickopen/common/quickOpenConfig.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as nls from 'vs/nls';
+import { Extensions, IConfigurationRegistry } from 'vs/platform/configuration/common/configurationRegistry';
+import { Registry } from 'vs/platform/registry/common/platform';
+
+export function registerQuickOpenConfiguration(): void {
+	const configurationRegistry = Registry.as<IConfigurationRegistry>(Extensions.Configuration);
+	configurationRegistry.registerConfiguration({
+		id: 'quickOpen',
+		order: 100,
+		title: nls.localize('TaskQuickOpenHistory', "Quick Open"),
```

Comment:
This setting is tasks specific, so it should be in task.contribution.ts in the `configurationRegistry` there. 
It follows then that the name of the setting should be task.quickOpen.history